src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -41,133 +40,98 @@ const propTypes = {
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    reimbursementAccount: {},
+};
+
```

Comment:
Not needed, since `reimbursementAccount` prop should be required

Reply:
done